packages/server/src/controllers/accounts.ts

```diff
@@ -1,14 +1,24 @@
 import { Context } from "koa";
 import { createOneAccount, getAccountById } from "../db/accounts";
-import { hashPassword } from "../utils/auth";
+import { hashPassword, createAccountSession } from "../utils/auth";
 
 export const createAccount = async (ctx: Context) => {
   try {
     const { firstName, lastName, email, password } = ctx.request.body;
 
     const hashedPassword = await hashPassword(password);
 
-    await createOneAccount(firstName, lastName, email, hashedPassword);
+    const [accountId] = await createOneAccount(
+      firstName,
+      lastName,
+      email,
+      hashedPassword,
+    );
+
+    const sessionId = await createAccountSession(accountId);
+
+    ctx.cookies.set("laperette_session", sessionId);
```

Comment:
move this cookie name in a constant, as you use it at least 2times

Reply:
Yes fair enough, like in an env variable ? Or in a separate file centralizing all the constants?